docs/docs/functions/misc.md

```diff
@@ -8,6 +8,29 @@ license: CC-BY-SA-4.0
 !!! note "Uncategorized"
     This page references all other functions that don't fit in the previous categories. Overtime, they might be moved to a different or new category.
 
+## ->
+
+Groups and executes multiple shards together (as if they were a single shard). Also called shard-container.
+
+This is useful for cases where you need to execute a number of shards but are allowed to execute only one: for example, as the return value of a function (see function `print-messages` below).  
```

Comment:
Again "execute" has a specific meaning. We aren't executing anything within `->`. Shards are only "activated" when a wire is scheduled on a mesh.

Reply:
Agreed, fixed.
(Rebased and force-pushed).